docs/getting-started/advanced-guide/cardinality.md

```diff
@@ -1,10 +1,12 @@
 # Cardinality
 
-This page will describe what cardinality is, how to add it to your Gaffer graph and how to use it in Gaffer queries.
+This page describes what cardinality is, how to add it to your Gaffer graph and how to use it in
```

Comment:
This level of line wrapping is too aggressive. I think we want an automatic linting setup to fix this all in one go. Making gradual changes as part of other PRs makes reviewing more difficult.

Reply:
Line wrap at 80-100 chars is kind of the go to standard length (this is 100). To wrap or not is a debate ([see](https://about.gitlab.com/blog/2016/10/11/wrapping-text/)) but if working within a text editor it makes it a lot cleaner to work with and review. I've edited this file as it came under the getting started section so just applied the standard wrap to be consistent with other files in this section.

Reply:
So I'm in the don't wrap camp 😆. Simply because horizontal scrolling is not an issue when you can soft wrap in the IDE.

You don't need to change this, but in future changing line wrap of existing text outside the scope of the PR should be avoided. If we want to change this universally across the repo that should be done in one go and automation used to manage in future.